Accessories/ExpansionBoard_uInverter/REV20220329B/README.md

```diff
@@ -0,0 +1,29 @@
+# uInverter Board
+
+This document describes the information necessary to place an order for the fabrication of the uInverter board.  
+
+This folder contains the compiled design outputs for the uInverter expansion board. This includes schematics, images of the PCB layout, the bill of materials (BOM) and the files needed to order fully populated boards. 
+
+## PCB Properties
+| Question          | Answer        |
+|-------------------|---------------|
+| Size              | 3.5" x 6.9"   |
+| Number of Layers  | 2             |
+| Number of components | 66         |
+|Number of holes    | 196           |
+| Thickness         | 1.7mm         |
+| Min Track/Spacing | 6/6mil          |
+| Min Hole Size     | 14mil           |
+| Silkscreen        | Top Only      |
+| Part Count        | 66            |
+
+
+## Connectors and connections
+
+Please check the [docs folder](../docs/) for more details on the connectors and pin-outs and other documentation.
```

Comment:
I believe this PR removes the `docs/` folder.

Please update this text to point the user to the `docs.amdc.dev` website for the uInverter section

Reply:
Also propagate this change to the `REV A` folder readme file

Reply:
I updated the landing page itself for uInverter to point to the docs.amdc.dev website's uInverter section.